packages/expo-screen-orientation/ios/ScreenOrientationUtilities.swift

```diff
@@ -0,0 +1,111 @@
+import ExpoModulesCore
+
+internal func isPad() -> Bool {
+  return UIDevice.current.userInterfaceIdiom == .pad
+}
+
+// https://medium.com/@cafielo/how-to-detect-notch-screen-in-swift-56271827625d
+internal var doesDeviceHaveNotch = {
+  var bottomSafeAreaInsetIsPositive = false
+  EXUtilities.performSynchronously {
+    bottomSafeAreaInsetIsPositive = (UIApplication.shared.delegate?.window??.safeAreaInsets.bottom ?? 0.0) > 0.0
+  }
+  return bottomSafeAreaInsetIsPositive
+}()
+
+extension UIInterfaceOrientationMask {
+  internal func toUIInterfaceOrientation() -> UIInterfaceOrientation {
+    switch self {
+    case .portrait: return .portrait
+    case .portraitUpsideDown: return.portraitUpsideDown
+    case .landscapeLeft: return .landscapeLeft
+    case .landscapeRight: return .landscapeRight
+    default: return .unknown
+    }
+  }
+
+  internal func contains(_ orientation: UIInterfaceOrientation) -> Bool {
+    // This is how the mask is created from the orientation
+    let maskFromOrientation = orientation.toInterfaceOrientationMask()
+    return self.contains(maskFromOrientation)
```

Comment:
```suggestion
    return contains(maskFromOrientation)
```
I have mixed feelings about it. Using the `self` keyword may not be essential, but it could enhance the readability of the code. It's ultimately your decision, but if you choose to delete it, please do so for all instances in this file.

Reply:
I think I prefer the version with self because this is an extension and it might not be obvious if we are referencing external function or a method right away (unless someone knows the syntax highlighting colors of xcode by heart 🤓). @tsapeta which version do you prefer?